src/test/groovy/com/stormpath/tck/forgot/ChangePasswordIT.groovy

```diff
@@ -168,12 +176,18 @@ class ChangePasswordIT extends AbstractIT {
     @Test(groups=["v100", "html"])
     public void changeRedirectsToErrorUriForInvalidSptokenWhenPosting() throws Exception {
 
-        given()
+        saveCSRFAndCookies(ForgotRoute)
+
+        def req = given()
             .accept(ContentType.HTML)
             .contentType(ContentType.URLENC)
             .queryParam("sptoken", "NOTEVENCLOSETOVALID")
             .param("password", "foobar123!")
-        .when()
+            .param("confirmPassword","foobar123!")
```

Comment:
@edjiang @robertjd @nbarbettini @bretterer - will providing the `confirmPassword` field hurt other integrations? I wouldn't think so, but I wanted to check. The backend password reset form and all of the Java Integrations require password confirmation.